models/Stc.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose');
+
+const StcSchema = mongoose.Schema({
```

Comment:
Peut-être utiliser `FinalPay` (https://www.breathehr.com/blog/how-to-calculate-an-employees-final-pay). Qu'en penses-tu ?

Reply:
Super idée ! Done